src/Toolkit/Toolkit/LocalizedStrings/Resources.resx.log.htm

```diff
@@ -0,0 +1,15 @@
+<html>
+<head>
+<script language="JavaScript" type="text/javascript">
+<!-- Copyright 2005, Sandeep Gangadharan -->
+<!-- For more free scripts go to http://www.sivamdesign.com/scripts/ -->
+<!--
+if (document.getElementById) {
+document.writeln('<style type="text/css"><!--')
+document.writeln('.texter {display:none} @media print {.texter {display:block;}}')
+document.writeln('//--></style>') }
+function openClose(theID) {
+if (document.getElementById(theID).style.display == "block") { document.getElementById(theID).style.display = "none" }
+else { document.getElementById(theID).style.display = "block" } }
+// -->
+</script></head>
```

Comment:
I don't think this file should be included in the PR

Reply:
@dotMorten Thank you for the comment.  We will review and create another commit.

Reply:
@dotMorten We have added a new commit to remove the Resources.resx.log.htm file: 
https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/commit/c9ace52ff3bdd55e270103f7a0e19266b0ca814f

Please review.